mmv1/third_party/terraform/services/bigquery/resource_bigquery_dataset_iam_member_test.go

```diff
@@ -62,3 +90,32 @@ resource "google_service_account" "bqviewer" {
 }
 `, datasetID, saID)
 }
+
+func testAccBigqueryDatasetIamMember_iamMember(datasetID, wifIDs string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset_iam_member" "access" {
+  dataset_id = google_bigquery_dataset.dataset.dataset_id
+  role       = "roles/viewer"
+  member     = "iamMember:principal://iam.googleapis.com/${google_iam_workload_identity_pool.wif_pool.name}/subject/test"
```

Comment:
API is returning
```
    {
      "role": "roles/viewer",
      "iamMember": "principal://iam.googleapis.com/projects/<project-num>/locations/global/workloadIdentityPools/<pool-id>/subject/test"
    }
```

Yet Terraform is looking for the exact string `iamMember:principal://iam.googleapis.com/projects/<project-num>/locations/global/workloadIdentityPools/<pool-id>/subject/test`

```
2024/02/09 19:12:01 [DEBUG]: Looking for member "iamMember:principal://iam.googleapis.com/projects/<project-num>/locations/global/workloadIdentityPools/<pool-id>/subject/test" in found binding
2024/02/09 19:12:01 [DEBUG]: Member "iamMember:principal://iam.googleapis.com/projects/<project-num>/locations/global/workloadIdentityPools/<pool-id>/subject/test" for binding for role "roles/viewer" with condition tpgiamresource.conditionKey{Description:"", Expression:"", Title:""} does not exist in policy of Bigquery Dataset ci-test-project-188019/<dataset-id>, removing from state.
2024-02-09T19:12:01.373Z [ERROR] vertex "google_bigquery_dataset_iam_member.access" error: Provider produced inconsistent result after apply
```

Is this just a matter of changing the test string?